spec.html

```diff
@@ -17134,9 +17134,9 @@ <h2>Syntax</h2>
     <emu-grammar type="definition">
       IfStatement[Yield, Await, Return] :
         `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return] `else` Statement[?Yield, ?Await, ?Return]
-        `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return]
+        `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return] [lookahead &ne; `else`]
```

Comment:
```suggestion
        `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return] [lookahead != `else`]
```
Change to "!=" for consistency with how it's expressed in other productions.

(But don't change it to "!=" in the Note, because that's not processed by grammarkdown.)

Reply:
I don't know what you're talking about here. Every use of lookahead in the spec is followed by &ne; or &#8713;.

Reply:
Not in the source.